src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverterSettings.cs

```diff
@@ -51,7 +51,7 @@ internal JsonClassInfo GetOrAddClass(Type classType)
             return result;
         }
 
-        public JsonReaderOptions ReaderOptions { get; set; }
+        internal JsonReaderOptions ReaderOptions { get; set; }
```

Comment:
I just marked it internal for now since this is consuming the source package on netstandard2.0, where ideally it would be in the package it self. We should revert this at that point and make it public again.